src/main/java/org/openstreetmap/atlas/geography/atlas/statistics/AtlasStatistics.java

```diff
@@ -76,7 +76,7 @@ public int hashCode()
         @Override
         public String toString()
         {
-            return this.tag + "," + this.type + "," + this.subType;
+            return this.tag + "," + this.type + "," + this.subType.replace("\"", "\"\"");
```

Comment:
Could [StringEscapeUtils](https://commons.apache.org/proper/commons-text/javadocs/api-release/org/apache/commons/text/StringEscapeUtils.html) be helpful here to cover more cases?

Reply:
Thanks for the suggestion! Double quotes are the only thing that needs to be escaped in CSV, but this will wrap strings with a newline or comma in double quotes as well. I'll make the change.